whisper_trt/model.py

```
@@ -524,6 +526,50 @@ def load(cls, trt_model_path: str) -> WhisperTRT:
         whisper_trt = whisper_trt.cuda().eval()
         return whisper_trt
 
+    @classmethod
+    @torch.no_grad()
+    def load_from_trt(
+        cls, engine_path: str, name: str, verbose: bool = False
+    ) -> WhisperTRT:
+        # 1) load the raw engine
+
+        trt_mod = torch2trt.TRTModule().cuda()
```

Pylint (3.3.7)

[info] Module 'torch2trt' has no 'TRTModule' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects.

(I1101)